draft-ietf-quic-applicability.md

```diff
@@ -207,11 +207,11 @@ of these issues.
 ## Session resumption versus Keep-alive {#resumption-v-keepalive}
 
 Because QUIC is encapsulated in UDP, applications using QUIC must deal with
-short idle timeouts. Deployed stateful middleboxes will generally establish
-state for UDP flows on the first packet state, and keep state for much shorter
-idle periods than for TCP. According to a 2010 study ({{Hatonen10}}), UDP
-applications can assume that any NAT binding or other state entry will be
-expired after just thirty seconds of inactivity.
+short network idle timeouts. Deployed stateful middleboxes will generally
+establish state for UDP flows on the first packet state, and keep state for
+much shorter idle periods than for TCP. According to a 2010 study
```

Comment:
Do we have any evidence that this recommendation is followed?  It seems safer to avoid mention of specifics for TCP, especially since they aren't really relevant to the current discussion.

Reply:
I believe there's strong evidence it's not followed.  I believe I discussed this with @MikeBishop at some point, but I don't know if there's any relevant public data?

Reply:
Taking this suggestion but editing it a bit to reflect state of the literature AIUI.

Reply:
Based on my vague memories of a study done at Microsoft several years ago, a 30-minute idle period on a TCP connection was tolerated by most firewalls, but beyond that you start finding your connection silently closed.